main.tf

```diff
@@ -125,6 +125,11 @@ resource "aws_iam_policy" "s3" {
   policy = join("", data.aws_iam_policy_document.s3.*.json)
 }
 
+data "aws_s3_bucket" "website" {
+  count  = local.enabled ? 1 : 0
```

Comment:
So is the website bucket name always required now? What happens if we run this without a website bucket?

Reply:
Added guards and tested with and without, works fine. After merging this PR CI will be fixed for the other PRs as well because of the `related` missing issue.